examples/mocha/scenarios/desktop/addons.spec.ts

```diff
@@ -4,8 +4,8 @@ import {expect} from 'chai';
 import type {Browser, BrowserContext, Page} from 'playwright';
 
 // Import browser helper
-import {browserHelper} from '~/src';
-import {GlobalVars} from '~/src';
+import {browserHelper} from '~/src/helpers/browserHelper';
+import {GlobalVars} from '~/src/helpers/globalVars';
```

Comment:
Since we have an index on `~/src` we don't need all the path, but what error is displayed here ?

Reply:
The duplicate import error was displayed.

Reply:
```suggestion
import {browserHelper, GlobalVars} from '~/src';
```
this should resolve the problem ?

Reply:
Yes, this resolve also the problem : I fix it.